src/seedu/addressbook/data/person/Name.java

```diff
@@ -9,12 +9,11 @@
  * Represents a Person's name in the address book.
  * Guarantees: immutable; is valid as declared in {@link #isValidName(String)}
  */
-public class Name {
+public class Name extends Contact{
```

Comment:
**Name is not a Contact** though it has similar logic to other types of Contacts. Activity clearly specifies that you need to extend _Phone_, _Email_ and _Address_ from _Contact_.